root/static/scripts/edit/URLCleanup.js

```diff
@@ -539,6 +539,42 @@ const CLEANUPS: CleanupEntries = {
       return {result: false, target: ERROR_TARGETS.URL};
     },
   },
+  'acum': {
+    match: [/^(https:\/\/)?nocs\.acum\.org\.il\/acumsitesearchdb\//i],
+    restrict: [LINK_TYPES.otherdatabases],
+    clean(url) {
+      return url.replace(/&.*/, '').replace('/version?', '/work?');
+    },
+    validate(url, id) {
+      const m = /^(https:\/\/)?nocs\.acum\.org\.il\/acumsitesearchdb\//i.test(url);
```

Comment:
Normally the point of `m` is to extract a part of the URL you're going to compare against. It would make sense to either rename this to "isAcumLink" or whatnot, or actually extract the bits you want to compare against here already.

Also, don't do `(https:\/\/)?` here - clean the URL up to always have the same (probably `https://`?) as part of the `clean` method above (we do that for many URLs, just copy it). Then just use the expected clean format here.

Reply:
Done.
I did the match here in the initial implementation but the regex got too complicated with all the different supported entities.